src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralReferenceGenerator.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service
+
+import org.springframework.stereotype.Component
+
+@Component
+class ReferralReferenceGenerator(
+  private val prefixChars: List<Char> = ('A'..'Z').toList().filterNot { it in listOf('I', 'O') },
+  private val numbers: List<Char> = ('0'..'9').toList(),
+) {
+  private val ambiguousPairs = listOf(
+    Pair("0", "O"),
```

Comment:
these first two aren't nessecary anymore, but it doesn't hurt to leave them here

Reply:
I think the postfix might still contain "O" or "I" outside our control; overall it feels better to leave it than leave the question hanging on "why not 0/O, 1/I", what do you think?

Reply:
sure thing 👍